models/reviews-parsing-ner-aspects/model.py

```diff
@@ -0,0 +1,69 @@
+import os
+import json
+from loguru import logger
+from mlserver import MLModel, types
+from mlserver.codecs import StringCodec
+from mlserver.types import InferenceRequest, InferenceResponse, ResponseOutput
+import mlflow
+
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+# Service name is required for most backends
+resource = Resource(attributes={SERVICE_NAME: "rpna-mlserver"})
+jaeger_url = os.environ.get("JAEGER_URL")
+
+traceProvider = TracerProvider(resource=resource)
+processor = BatchSpanProcessor(OTLPSpanExporter(endpoint=f"{jaeger_url}/v1/traces"))
+traceProvider.add_span_processor(processor)
+trace.set_tracer_provider(traceProvider)
+tracer = trace.get_tracer(__name__)
+
+
```

Comment:
Code này có vẻ chưa đc format thì phải =)) sếp nghiên cứu thử black vs isort để format code tí nha

Reply:
code đó em format bằng black 24.4.2 rồi đó nhưng đúng là em k dùng isort. Vì em có convention riêng khi define imports: Built-in > General Packages > Niche Packages > In-repo Modules. Qua đây cũng muốn hỏi luôn là ở các chỗ sếp làm họ có hay dùng isort k? @quan-dang

Reply:
Có nhé sếp ơi